Lib/xml/etree/ElementTree.py

```diff
@@ -1256,8 +1256,9 @@ def __del__(self):
                 source.close()
 
     it = IterParseIterator()
+    it.root = None
     wr = weakref.ref(it)
-    del IterParseIterator
+    del iterator, IterParseIterator
```

Comment:
Given that w/o `iterator` deletion benchmarks were OK, I would consider deleting this line entirely. Yes, it is harmless, but it is also useless.

Reply:
I was afraid that the reference to `IterParseIterator` could be kept after returning from the function, and it could complicate the reference graph. But it seems that they are cleared before the generator function starts executing. I think it is safe to remove this `del`.

Reply:
Ok, I'll remove the `del`.